### ISSUE TITLE
First time with this tool... is it at all possible to open .eml file using smtp4dev?

### ISSUE THREAD

**Issue by skdutta-ga** [rnwood/smtp4dev#555]

**Comment by metalrise** [comment]
Yes, after you click on a message there is a Download button that saves the email in eml format

**Comment by skdutta-ga** [comment]
I was asking the opposite. I have an eml file, I would like to open it in smtp4dev to check out its headers etc.

**Comment by rnwood** [comment]
Hi. At the moment it's not possible to import from `.eml` file for inspection. The app can only view messages that have been received via SMTP. 

You could workaround this by sending the .eml file you have over SMTP to smtp4dev (although the client may add/modify the headers):
```
cat file.eml | sendmail -t
```
(on Windows use https://github.com/sendmail-tls1-2/main to get sendmail)

Marking this as a feature request as may consider implementing in future.

**Comment by rnwood** [comment]
Implemented in #1830
